tests/migration/migration.go

```diff
@@ -31,6 +31,8 @@ import (
 	"sync"
 	"time"
 
+	"kubevirt.io/api/migrations/v1alpha1"
```

Comment:
please move this in the proper import section below

Reply:
Just for the record, I have to say I'm pretty hesitant on that. I think that it's something that either `make fmt` should handle or our linter should fail build for, but I don't really think it should be a part of the review process. IMHO it's an unnecessary burden on developers that we should either automate or ignore.

Either way, I moved it.

Reply:
Totally agree about the automation :)
Just FYI I'm just following the coding conventions: https://github.com/kubevirt/kubevirt/blob/main/docs/coding-conventions.md

Reply:
No worries, my pleasure :)